source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfigPrivate.h

```diff
@@ -6,13 +6,24 @@
 //
 #import "ACOHostConfig.h"
 #import "HostConfig.h"
+#import "TextBlock.h"
+#import <UIKit/UIKit.h>
 
 using namespace AdaptiveCards;
 
 @interface ACOHostConfig()
 
 - (instancetype)initWithConfig:(std::shared_ptr<HostConfig> const &)config;
-- (std::shared_ptr<HostConfig>)getHostConfig;
+- (std::shared_ptr<HostConfig> const &)getHostConfig;
```

Comment:
>const & [](start = 31, length = 7)

Why did you change this to return by reference? That seems unsafe.

Reply:
thanks

---
In reply to: [171736471](https://github.com/Microsoft/AdaptiveCards/pull/1178#discussion_r171736471) [](ancestors = 171736471)